README.md

```diff
@@ -27,6 +27,11 @@ integration from the Jetpack Benchmark library.
 This sample compares how different ViewGroups
 (ConstraintLayout vs traditional layouts) affect the UI performance.
 
+## [MacrobenchmarkSample](MacrobenchmarkSample)
```

Comment:
Move the "remember to checkout submodules" bit above to be under BenchmarkNdkSample, since it only applies to that, since as written now it looks like it applies to all of them.

Can also either a) just remove `./gradlew benchmark:cC` above, or b) duplicate it for each of the 3 benchmark samples (`benchmark:cC` for the first two, `macrobenchmark:cC` for the last)